README.adoc

```diff
@@ -0,0 +1,94 @@
+= C++ example project scanned on SonarCloud using Travis CI
```

Comment:
I would emphasize here (in the title or just below it) that the project uses gcov because otherwise, it's not apparent at first glance.

Reply:
Added.